summariser/src/frame.rs

```diff
@@ -5,11 +5,17 @@ use thiserror::Error;
 
 #[derive(Error, Debug)]
 pub enum LoadError {
-    #[error("No series in result: {result:?}")]
-    NoSeriesInResult { result: serde_json::Value },
+    #[error("No series in table {table}: {result:?}")]
+    NoSeriesInResult {
+        table: String,
```

Comment:
currently this error didn't provide enough information to find out what was wrong.